### DISCUSSION TITLE
Why the mean stomatal conductance always low?

### DISCUSSION THREAD

**Discussion by LianGong98** [NGEET/fates#1037] [Q&A]
Hi, FATES developer
 I run FATES in BCI using the fundamental stomatal conductance theory (Ball-Berry model), the output of the mean stomatal conductance (umol m-2 s-1) seems low? 
If I multiply the conversion factor micromoles per mole (1/umol_per_mol), the rage of the mean stomatal conductance always less than 0.1 mol m-2 s-1. And I suspect the lower conductance resulted in the canopy transpiration (mm/s) less than observation. I do not know how to solve this problem and I hope to get help.
![C_STOMATA_in_bci-test-hydro-3 C63f3ea8-F3f6749d 2023-06](https://github.com/NGEET/fates/assets/97998383/50177d7d-6091-49fc-a4e3-d320103b6cfc)
Any help would be greatly appreciated!

**Comment by rgknox** [comment]
@LianGong98 is the GPP also low? If this is so, I would first look at photosynthesis parameters. The conductance of water should also scale with the conductance of CO2, and may respond to parameters like vcmax.   If the GPP is not low, I would look at things that control btran.  For a non-hydro version of FATES, we have these two parameters:
[fates_nonhydro_smpcsc and fates_nonhydro_smpso](https://github.com/NGEET/fates/blob/main/parameter_files/fates_params_default.cdl#L431-L436).  You could try lowering those values.

**Reply by LianGong98** [reply]
Thank you very much,and I will check the btran now.
